index.html

```diff
@@ -1012,6 +1012,92 @@ <h3>
           conventions or limitations of the host operating system.
         </p>
       </section>
+      <section>
+        <h3>
+          `file_handlers` member
+        </h3>
+        <p>
+          The [=manifest's=] <code><dfn data-export="" data-dfn-for=
+          "manifest">`file_handlers`</dfn></code> member is a [=list=] that
+          represents urls within the app that can handle launches of a given
+          file type (TODO: link to definition of file type).
+        </p>
+        <p>
+          How file handlers are registered and disambiguated are at the
+          discretion of the operating system.
+        </p>
+        <p>
+          To <dfn>process the `file_handlers` member</dfn>, given [=ordered
+          map=] |json:ordered map|, [=ordered map=] |manifest:ordered map|:
+        </p>
+        <ol class="algorithm">
+          <li>Let |processedFileHandlers:list| be a new [=list=].
+          </li>
+          <li>Set |manifest|["file_handlers"] to |processedFileHandlers|.
+          </li>
+          <li>If |json|["file_handlers"] doesn't [=map/exist=] or
+          |json|["file_handlers"] is not a [=list=], return.
+          </li>
+          <li>[=list/For each=] |entry:ordered map| of |json|["file_handlers"]:
+            <ol>
+              <li>Let |file_handler:ordered map| be [=process a file handler
+              item=] with |entry|.
+              </li>
+              <li>If |file_handler| is failure, continue.
+              </li>
+              <li>[=list/Append=] |file_handler| to |processedFileHandlers|.
+              </li>
+            </ol>
+          </li>
+        </ol>
+        <p>
+          A user agent SHOULD register the file handlers with the operating
+          system via interactions that are consistent with:
+        </p>
+        <ul>
+          <li>The [=file handler/icon=] and [=file handler/name=] appearing in
+          an "Open with.." menu in the context menu of associated files.
+          </li>
+          <li>The [=file handler/icon=] appearing as the icon of associated
+          files.
+          </li>
+          <li>File types with multiple possible handlers can be disambiguated
+          by UX provided by the operating system.
+          </li>
+        </ul>
+        <p>
+          A user agent MAY (TODO: should?) prevent registered file handlers
+          from registering as the 'default' file handler for a give type if the
+          operating system allows.
+        </p>
+      </section>
+      <section>
+        <h3>
+          Privacy consideration: Default [=file handler=].
+        </h3>
+        <p>
+          Depending on the operating system capabilities that the
+          [=manifest/file handler=] could become a 'default' handler (e.g.
+          handling launches of a given file type automatically) of a given file
+          type (todo define? link?) or [=mime type=] without the explicit
```

Comment:
Maybe go ahead and create a stub definition for "file type" in this doc, and put a TODO there, then have these links point at it?